src/components/Calculator.test.js

```diff
@@ -0,0 +1,29 @@
+import { render, screen, act } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import renderer from "react-test-renderer";
+
+import Calculator from "./Calculator";
+
```

Comment:
[REQUIRED] Kindly use React Testing Library to simulate user interaction and ensure that you test all functionalities and scenarios in operate.js and calculate.js like addition, and subtraction among others.